crates/cdk-signatory/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "cdk-signatory"
+version = "0.6.0"
+edition = "2021"
+description = "CDK signatory default implementation"
+
+[features]
+default = []
+server = []
+
+[dependencies]
+async-trait = "0.1.83"
+bitcoin = { version = "0.32.2", features = [
+    "base64",
+    "serde",
+    "rand",
+    "rand-std",
+] }
+cdk-common = { path = "../cdk-common", default-features = false, features = [
+    "mint",
+] }
+tracing = "0.1.41"
+tokio = { version = "1.21", features = ["full"] }
+tonic = "0.12.3"
+prost = "0.13.4"
```

Comment:
These break MSRV, I'll figure that out on the management-rpc [branch](https://github.com/cashubtc/cdk/pull/543) and report back